README-Mandrel.md

```diff
@@ -0,0 +1,45 @@
+# Mandrel
+
+Mandrel is a downstream distribution of the GraalVM community edition.
+Mandrel's main goal is to provide a `native-image` in-place replacement to be used by [Quarkus](https://quarkus.io).
```

Comment:
@maxandersen @adinn please review ^^

Reply:
I'm really not keen on saying mandrel replaces Quarkus or even just some part of Quarkus. I don't think Oracle will be happy with that idea either.

Reply:
I would prefer if Thomas Qvarnstrom decided what to say here.

Reply:
@adinn I guess you mean GraalVM instead of Quarkus. Thanks for your comments.
Summoning @tqvarnst for a review :)

Reply:
I agree with @adinn here. Maybe we could say something like this instead: "Mandrel's main goal is to align the `native-image` capabilities from GraalVM with OpenJDK and Red Hat Enterprise Linux libraries to improve maintainability for native Quarkus applications."

Reply:
Can we just change this to say "Mandrel's main goal is to provide a `native-image` release specifically to support [Quarkus](https://quarkus.io)."

Reply:
Of course, that doesn't explain *why* we want a different release to GraalVM and doesn't explain that we are happy for Quarkus to be used with GraalVM releases.

Reply:
Ah, didn't see Thomas's comment

"Mandrel's main goal is to provide a native-image release specifically to support Quarkus. The aim is to align the native-image capabilities from GraalVM with OpenJDK and Red Hat Enterprise Linux libraries to improve maintainability for native Quarkus applications."

Reply:
How does that sound @tqvarnst?

Reply:
Yes, that sounds good.

Reply:
Ok, we have a wrap!